process/mock/addressConverterStub.go

```diff
@@ -35,7 +35,10 @@ func (acs *AddressConverterStub) PrepareAddressBytes(addressBytes []byte) ([]byt
 
 // AddressLen -
 func (acs AddressConverterStub) AddressLen() int {
```

Comment:
add pointer receiver *AddressConverterStub, and maybe rename AddressLenHandler to AddressLenCalled to be in the same format with the other methods in this file

Reply:
done.